lib/workload/stateless/stacks/cttso-v2-pipeline-manager/deploy/constructs/cttsov2-icav2-manager/index.ts

```diff
@@ -194,7 +194,7 @@ export class Cttsov2Icav2PipelineManagerConstruct extends Construct {
       generate_trimmed_samplesheet_lambda_obj,
       upload_samplesheet_to_cache_dir_lambda_obj,
     ].forEach((lambda_obj) => {
-      lambda_obj.grantInvoke(<iam.IRole>stateMachine.role);
+      lambda_obj.currentVersion.grantInvoke(<iam.IRole>stateMachine.role);
```

Comment:
@williamputraintan @mmalenic @raylrui FYI, folks. 

If we use `currentVersion` prop of Lambda, we can get away of cdk generating `*` to ARN. This is for cdk-nag `AwsSolutions-IAM5` suppression.